core/src/main/scala/org/apache/spark/deploy/mesos/MesosDriverDescription.scala

```diff
@@ -40,24 +41,28 @@ private[spark] class MesosDriverDescription(
     val cores: Double,
     val supervise: Boolean,
     val command: Command,
-    val schedulerProperties: Map[String, String],
+    schedulerProperties: Map[String, String],
```

Comment:
I'm probably missing your point but would that affect `val`? it might be `private` too but this change just removes `val`.